R/utils.R

```diff
@@ -8,14 +8,39 @@
 #' @usage lhs \%>\% rhs
 NULL
 
-at_selection <- function(nm, .at){
-  if (is_quosures(.at)) {
-    lifecycle::deprecate_warn("1.0.0", I("using `vars()` in .at"))
+at_selection <- function(x, at, error_arg = caller_arg(at), error_call = caller_env()) {
+  if (is_formula(at)) {
```

Comment:
`is_formula()` returns `TRUE` for quosures. Do we have tests for tidyselect features?

Reply:
We do, so this took me a minute to figure out — `vars` is a `<list_of<quosure>>` not a `quosure`
`

Reply:
Ah that makes sense.